grafana-plugin/src/containers/GSelect/GSelect.tsx

```diff
@@ -54,18 +55,15 @@ const GSelect = observer((props: GSelectProps) => {
     displayField = 'display_name',
     valueField = 'id',
     isMulti = false,
-    nullItemName,
-    dropdownRender,
     getOptionLabel,
     showWarningIfEmptyValue = false,
     getDescription,
+    excludedOptionsIdList,
```

Comment:
I think it's better to add prop like:
```
filterOptions?: <T>(item: SelectableValue<T>) => boolean;

```

it allows us to filter options in more flexible way, instead of checking if item.id is present in `excludedOptionsIdList` array we can check every item in filter function, what do you think?

Reply:
Good idea, I'll change it to this instead.